javascript-and-json/main.js

```diff
@@ -0,0 +1,33 @@
+var books = [
+  {
+    isbn: '400',
+    author: 'Quinn Russell',
+    title: 'Book'
+  },
+  {
+    isbn: '500',
+    author: 'Brett',
+    title: 'The Teacher Guy'
+  },
+  {
+    isbn: '600',
+    author: 'Ema Smith',
+    title: 'That Girl'
+  }
+];
+
+console.log('Book Array', books, typeof (books));
+
+// stringify()
+var booksJSON = JSON.stringify(books);
+console.log('JSON books', booksJSON, typeof (booksJSON));
+
+// JSON student obj as string
+var student = JSON.stringify(
```

Comment:
you don't want to use `.stringify` here. You want to manually create a JSON string